content/en/flux/installation/bootstrap/azure-devops.md

```diff
@@ -51,29 +51,29 @@ inside the `flux-system` namespace.
 
 {{% alert color="info" title="Token rotation" %}}
 Note that Azure DevOps PAT have an expiry date. To rotate the token before it expires,
-delete the `flux-system` secret from the cluster and re-run
-the bootstrap command using a valid PAT.
+delete the `flux-system` secret from the cluster and create a new one with the new PAT:
+
+```sh
+flux create secret git flux-system \
+   --url=https://dev.azure.com/<org>/<project>/_git/<repository> \
+   --username=git \
+   --password=$AZURE_DEVOPS_PAT
```

Comment:
This environment variable hasn't been used prior on this page but `<az-token>` so maybe use that for consistency's sake?
```suggestion
   --password=<az-token>
```